src/main/java/org/openstreetmap/atlas/checks/base/CheckResourceLoader.java

```diff
@@ -64,6 +71,12 @@ public CheckResourceLoader(final Configuration configuration)
         this.packages = Collections.unmodifiableSet(Iterables.asSet((Iterable<String>) configuration
                 .get("CheckResourceLoader.scanUrls", Collections.singletonList(DEFAULT_PACKAGE))
                 .value()));
+        final Map<String, List<String>> groups = configuration.get("groups", Collections.emptyMap())
+                .value();
+        groups.keySet().forEach(group ->
+        {
+            groups.get(group).forEach(country -> this.countryGroups.add(country, group));
```

Comment:
Just curious - what is the value of using `MultiMap` over `Map<String, List<String>>`?

Reply:
I don't have to handle initializing the list when I add key value pairs